packages/pharos-site/src/pages/content-style-guide/grammar-and-style/index.mdx

```diff
@@ -10,7 +10,7 @@ import BestPractices from '../../../components/statics/BestPractices.tsx';
     }}
   >
     <p>
-      This is our house style; for anything not covered here we follow the
+      This is our house style; for anything not covered here we follow the&nbsp;
```

Comment:
I think we want it to be a normal ol' breakable space, unless I'm missing anything?

```suggestion
      This is our house style; for anything not covered here we follow the{' '}
```